### PR TITLE
Add user-facing RBAC roles

### LINKED ISSUE #305: Add RBAC roles associated with HMC management

### PR DESCRIPTION
**Pull request by Kshatrix**
Resolves #305 #306

### CODE REVIEW

File: templates/provider/hmc/templates/rbac/controller/roles.yaml [hunk 1]
```diff
@@ -55,13 +39,6 @@ rules:
   resources:
   - managements
   verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
```
Comment (eromanova) [inline templates/provider/hmc/templates/rbac/controller/roles.yaml:64]:
```
{{ include "rbac.editorVerbs" . | nindent 4 }}
```
is missing?

Comment (Kshatrix) [reply templates/provider/hmc/templates/rbac/controller/roles.yaml:64]: no, editorVerbs is wider than needed in this case

Comment (eromanova) [reply templates/provider/hmc/templates/rbac/controller/roles.yaml:64]: But we should still allow the controller to get/watch/update/etc. `Management` object. Now, the `verbs` are empty. Or am I missing something?

Comment (Kshatrix) [reply templates/provider/hmc/templates/rbac/controller/roles.yaml:64]: ah, this one is definitely missing, thanks

**Review by eromanova — APPROVED** [review]

**Pull request merged**